lib/engine/game/base.rb

```diff
@@ -2052,8 +2052,12 @@ def priority_deal_player
         end
       end
 
-      def next_sr_position(entity)
-        player_order = @round.current_entity&.player? ? @round.pass_order : @players
+      def next_sr_position(entity, order)
```

Comment:
The game class knows the order, so you don't need to pass it in. Call next_sr_player_order directly from here.